contrib/backup/functions

```diff
@@ -54,9 +56,23 @@ function backup_db () {
     ln -sfn ${BACKUP_DIR}/${TIMESTAMP}_zammad_db.mysql.gz ${BACKUP_DIR}/latest_zammad_db.mysql.gz
   elif [ "${DB_ADAPTER}" == "postgresql" ]; then
     echo "creating postgresql backup..."
-    su -c "pg_dump --no-privileges --no-owner -c ${DB_NAME} | \
-      gzip > /tmp/${TIMESTAMP}_zammad_db.psql.gz" postgres
-    mv /tmp/${TIMESTAMP}_zammad_db.psql.gz ${BACKUP_DIR}
+
+    export PGPASSFILE=$(mktemp)
+    chmod 600 "$PGPASSFILE"
+    cat <<EOT > "$PGPASSFILE"
+*:*:${DB_NAME}:${DB_USER}:${DB_PASS}
+EOT
+
+    PGPASSWORD="${DB_PASS}" pg_dump \
```

Comment:
Either use PGPASSWORD or PGPASSFILE, both should not be needed.

Reply:
🙈

Reply:
`PGPASSWORD` is removed in the current version.